albatross/covariance_functions/polynomials.h

```diff
@@ -28,17 +28,15 @@ struct ConstantTerm {};
  * underlying constant term would actually be a biased estimate
  * of the mean.
  */
-class Constant : public CovarianceTerm {
+class Constant : public CovarianceFunction<Constant> {
 public:
-  Constant(double sigma_constant = 10.) {
+  Constant(double sigma_constant = 10.) : name_("constant") {
```

Comment:
NIT suggestions: This and the initializer constant in the diagonal covariance class could be a named `#define` or constexpr just so that the numbers don't get lost along the way, and changing them at the top of the file is plain to see. I guess nobody should ever be using these classes without an argument anyway so it probably doesn't matter.

Reply:
Yeah, I'm not super satisfied with the `name_` member approach.  you have any opinions on an approach for `constexpr` strings along these lines: https://akrzemi1.wordpress.com/2017/06/28/compile-time-string-concatenation/